integration-tests/actions/vrf/vrfv2plus/setup_steps.go

```diff
@@ -235,51 +232,100 @@ func setupVRFNode(contracts *vrfcommon.VRFContracts, chainID *big.Int, config *v
 	nodeConfig := node.NewConfig(vrfNode.CLNode.NodeConfig,
 		node.WithKeySpecificMaxGasPrice(vrfNode.TXKeyAddressStrings, *config.CLNodeMaxGasPriceGWei),
 	)
-	l.Info().Msg("Restarting Node with new sending key PriceMax configuration")
+	l.Info().
+		Strs("Sending Keys", vrfNode.TXKeyAddressStrings).
+		Int64("Price Max Setting", *config.CLNodeMaxGasPriceGWei).
+		Msg("Restarting Node with new sending key PriceMax configuration")
 	err = vrfNode.CLNode.Restart(nodeConfig)
 	if err != nil {
 		return fmt.Errorf(vrfcommon.ErrGenericFormat, vrfcommon.ErrRestartCLNode, err)
 	}
 	return nil
 }
 
-func SetupVRFV2PlusWrapperEnvironment(
+func SetupVRFV2PlusWrapperForExistingEnv(
 	ctx context.Context,
+	sethClient *seth.Client,
+	vrfContracts *vrfcommon.VRFContracts,
+	vrfv2PlusTestConfig types.VRFv2PlusTestConfig,
+	consumerContractsAmount int,
 	l zerolog.Logger,
+) (*VRFV2PlusWrapperContracts, *big.Int, error) {
+	config := *vrfv2PlusTestConfig.GetVRFv2PlusConfig()
+	var wrapper contracts.VRFV2PlusWrapper
+	var err error
+	if *config.ExistingEnvConfig.UseExistingWrapper {
```

Comment:
in my opinion, it will be not obvious 

@kidambisrinivas wdyt?

Reply:
thanks for elaborate comment, now I understand your concern more clearly!

Reply:
Yes, I agree with @leeyikjiun here. Since we are using wrapper from existing env, let's use `useExistingEnv` to toggle between new and existing env and have an error/validation for WrapperAddress not empty in this case

Reply:
lets address it in a separate PR